radar/infrastructure_config/msmq.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+name: MSMQ
```

Comment:
I believe this belongs in data-management.

Reply:
Since this is not a place to store data I suggest it stays under infrastructure.

Reply:
My take on this is that `Infrastructure and Config` is about tech to deal with infrastructure, provisioning and CI. I think MSMQ makes more sense in `Data Management` as it is about application data flows. The category doesn't imply that data must be stored. I'd for sure say a system like RabbitMQ would belong in `Data Management`.

Reply:
Ok, moving it.